ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1280,6 +1280,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-sdn
+- as: aws-ipi-registry-s3-cloudfront-p2-f14
```

Comment:
To make the test items in order, I think it should be placed behind `as: aws-ipi-proxy-workers-rhel8-etcd-encryptio-p3-f28-destructive`

Reply:
If the testing is not critical enough, I suggest to set it on `f28`, and if we also cover it in arm ci jobs, that means, for 4.13 testing, the testing will be executed once per 2 weeks.

Reply:
@jianlinliu Updated, thanks for the advice